vraygolaem.cpp

```diff
@@ -884,6 +887,12 @@ void VRayGolaem::updateVRayParams(TimeValue t)
 
 	// vray
 	_frameOffset = pblock2->GetInt(pb_frame_offset, t);
+	const TCHAR *defaultMat_wstr=pblock2->GetStr(pb_default_material, t);
+	if (!defaultMat_wstr) _vrsceneFile="";
```

Comment:
Why do you set the _vrsceneFile member to "" here? Should it not be _defaultMaterial?